### ISSUE TITLE
Is it possible to monitor the input channels of an ASIO device?

### ISSUE THREAD

**Issue by jfrank14** [naudio/NAudio#359]
I have the requirement to show a UI with the volume levels of an ASIO recording device, and it would be ideal to do this without having to be recording anything. This is so that someone can set up for recording and speak into a microphone to test the connection.

Is this possible? Thanks.

**Comment by markheath** [comment]
Yes, although NAudio uses the terminology of "recording", actually all you're doing is capturing audio from the device, and its up to you whether you record that to disk, or just examine the values of each sample

**Comment by jfrank14** [comment]
So are you saying that to do what I want to do, I'd do something like `asioOut.InitRecordAndPlayback(null, channelCount, 44100);` exactly as if I were recording, except that I'd use the samples to show the input levels rather than anything else?

**Comment by markheath** [comment]
yes, it's quite a common way of working with input devices
